<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesEditor</class>
 <widget class="QMainWindow" name="NotesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_noteTitle"/>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWebEngineView" name="webView" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_noteLink">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_editNoteLink">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuF_ormat">
    <property name="title">
     <string>F&amp;ormat</string>
    </property>
    <widget class="QMenu" name="menuSt_yle">
     <property name="title">
      <string>St&amp;yle</string>
     </property>
     <addaction name="actionStyleParagraph"/>
     <addaction name="actionStyleHeading1"/>
     <addaction name="actionStyleHeading2"/>
     <addaction name="actionStyleHeading3"/>
     <addaction name="actionStyleHeading4"/>
     <addaction name="actionStyleHeading5"/>
     <addaction name="actionStyleHeading6"/>
     <addaction name="actionStylePreformatted"/>
     <addaction name="actionStyleAddress"/>
    </widget>
    <widget class="QMenu" name="menu_Align">
     <property name="title">
      <string>&amp;Align</string>
     </property>
     <addaction name="actionFormatAlignLeft"/>
     <addaction name="actionFormatAlignCenter"/>
     <addaction name="actionFormatAlignRight"/>
     <addaction name="actionFormatAlignJustify"/>
    </widget>
    <addaction name="menuSt_yle"/>
    <addaction name="menu_Align"/>
    <addaction name="separator"/>
    <addaction name="actionFormatBold"/>
    <addaction name="actionFormatItalic"/>
    <addaction name="actionFormatUnderline"/>
    <addaction name="actionFormatStrikethrough"/>
    <addaction name="separator"/>
    <addaction name="actionFormatIncreaseIndent"/>
    <addaction name="actionFormatDecreaseIndent"/>
    <addaction name="separator"/>
    <addaction name="actionFormatNumberedList"/>
    <addaction name="actionFormatBulletedList"/>
    <addaction name="separator"/>
    <addaction name="actionFormatFontName"/>
    <addaction name="actionFormatFontSize"/>
    <addaction name="separator"/>
    <addaction name="actionFormatTextColor"/>
    <addaction name="actionFormatBackgroundColor"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditUndo"/>
    <addaction name="actionEditRedo"/>
    <addaction name="separator"/>
    <addaction name="actionEditCut"/>
    <addaction name="actionEditCopy"/>
    <addaction name="actionEditPaste"/>
    <addaction name="separator"/>
    <addaction name="actionEditSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionInsertImage"/>
    <addaction name="actionCreateLink"/>
    <addaction name="actionInsertHtml"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuF_ormat"/>
  </widget>
  <widget class="QToolBar" name="standardToolBar">
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="actionEditUndo"/>
   <addaction name="actionEditRedo"/>
   <addaction name="separator"/>
   <addaction name="actionEditCut"/>
   <addaction name="actionEditCopy"/>
   <addaction name="actionEditPaste"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
  </widget>
  <widget class="QToolBar" name="formatToolBar">
   <property name="windowTitle">
    <string>Formatting</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionFormatBold"/>
   <addaction name="actionFormatItalic"/>
   <addaction name="actionFormatUnderline"/>
   <addaction name="actionFormatStrikethrough"/>
   <addaction name="separator"/>
   <addaction name="actionFormatAlignLeft"/>
   <addaction name="actionFormatAlignCenter"/>
   <addaction name="actionFormatAlignRight"/>
   <addaction name="actionFormatAlignJustify"/>
   <addaction name="separator"/>
   <addaction name="actionFormatDecreaseIndent"/>
   <addaction name="actionFormatIncreaseIndent"/>
   <addaction name="separator"/>
   <addaction name="actionFormatNumberedList"/>
   <addaction name="actionFormatBulletedList"/>
   <addaction name="separator"/>
   <addaction name="actionInsertImage"/>
   <addaction name="actionCreateLink"/>
   <addaction name="actionInsertHtml"/>
  </widget>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/document-save.png</normaloff>:/icons/22x22/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/document-new.png</normaloff>:/icons/22x22/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionEditUndo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-undo.png</normaloff>:/icons/22x22/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-redo.png</normaloff>:/icons/22x22/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionEditCut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-cut.png</normaloff>:/icons/22x22/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionEditCopy">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-copy.png</normaloff>:/icons/22x22/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEditPaste">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-paste.png</normaloff>:/icons/22x22/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionEditSelectAll">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/edit-select-all.png</normaloff>:/icons/22x22/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select A&amp;ll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFormatBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-text-bold.png</normaloff>:/icons/22x22/format-text-bold.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionFormatItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-text-italic.png</normaloff>:/icons/22x22/format-text-italic.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionFormatUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-text-underline.png</normaloff>:/icons/22x22/format-text-underline.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFormatStrikethrough">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-text-strikethrough.png</normaloff>:/icons/22x22/format-text-strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Strikethrough</string>
   </property>
  </action>
  <action name="actionFormatAlignLeft">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-justify-left.png</normaloff>:/icons/22x22/format-justify-left.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Left</string>
   </property>
  </action>
  <action name="actionFormatAlignCenter">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-justify-center.png</normaloff>:/icons/22x22/format-justify-center.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Center</string>
   </property>
  </action>
  <action name="actionFormatAlignRight">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-justify-right.png</normaloff>:/icons/22x22/format-justify-right.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Right</string>
   </property>
  </action>
  <action name="actionFormatAlignJustify">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-justify-fill.png</normaloff>:/icons/22x22/format-justify-fill.png</iconset>
   </property>
   <property name="text">
    <string>Align &amp;Justify</string>
   </property>
  </action>
  <action name="actionFormatIncreaseIndent">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-indent-more.png</normaloff>:/icons/22x22/format-indent-more.png</iconset>
   </property>
   <property name="text">
    <string>I&amp;ncrease Indent</string>
   </property>
  </action>
  <action name="actionFormatDecreaseIndent">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-indent-less.png</normaloff>:/icons/22x22/format-indent-less.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Decrease Indent</string>
   </property>
  </action>
  <action name="actionFormatBulletedList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-list-unordered.png</normaloff>:/icons/22x22/format-list-unordered.png</iconset>
   </property>
   <property name="text">
    <string>Bulle&amp;ted List</string>
   </property>
  </action>
  <action name="actionFormatNumberedList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/format-list-ordered.png</normaloff>:/icons/22x22/format-list-ordered.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Numbered List</string>
   </property>
  </action>
  <action name="actionInsertImage">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/insert-image.png</normaloff>:/icons/22x22/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Insert &amp;Image...</string>
   </property>
  </action>
  <action name="actionCreateLink">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/insert-link.png</normaloff>:/icons/22x22/insert-link.png</iconset>
   </property>
   <property name="text">
    <string>Create Link...</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/zoom-out.png</normaloff>:/icons/22x22/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/zoom-in.png</normaloff>:/icons/22x22/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStyleParagraph">
   <property name="text">
    <string>&amp;Paragraph</string>
   </property>
  </action>
  <action name="actionStyleHeading1">
   <property name="text">
    <string>Heading &amp;1</string>
   </property>
  </action>
  <action name="actionStyleHeading2">
   <property name="text">
    <string>Heading &amp;2</string>
   </property>
  </action>
  <action name="actionStyleHeading3">
   <property name="text">
    <string>Heading &amp;3</string>
   </property>
  </action>
  <action name="actionStyleHeading4">
   <property name="text">
    <string>Heading &amp;4</string>
   </property>
  </action>
  <action name="actionStyleHeading5">
   <property name="text">
    <string>Heading &amp;5</string>
   </property>
  </action>
  <action name="actionStyleHeading6">
   <property name="text">
    <string>Heading &amp;6</string>
   </property>
  </action>
  <action name="actionStylePreformatted">
   <property name="text">
    <string>Pre&amp;formatted</string>
   </property>
  </action>
  <action name="actionStyleAddress">
   <property name="text">
    <string>&amp;Address</string>
   </property>
  </action>
  <action name="actionFormatFontName">
   <property name="text">
    <string>&amp;Font Name...</string>
   </property>
  </action>
  <action name="actionFormatTextColor">
   <property name="text">
    <string>Text &amp;Color...</string>
   </property>
  </action>
  <action name="actionFormatBackgroundColor">
   <property name="text">
    <string>Bac&amp;kground Color...</string>
   </property>
  </action>
  <action name="actionFormatFontSize">
   <property name="text">
    <string>Font Si&amp;ze...</string>
   </property>
  </action>
  <action name="actionInsertHtml">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/22x22/insert-html.png</normaloff>:/icons/22x22/insert-html.png</iconset>
   </property>
   <property name="text">
    <string>Insert HTML...</string>
   </property>
   <property name="toolTip">
    <string>Insert HTML</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
